proto/channel.proto

```diff
@@ -4,4 +4,8 @@ package pb;
 
 message Channel {
     bytes public_key = 1;
+    string name = 2;
+    string description = 3;
```

Comment:
we may want a short description, limited to a reasonable number of characters and no markup, as well as an unbounded (well, bounded by claim size) field that would be shown on the full profile view

Reply:
regarding: https://github.com/lbryio/types/pull/17#issuecomment-463418314 - I think Name could be the channels real name, or whatever they want to nickname their channel...i.e. my channel is @iloveLBRY, but the name would be "I Love LBRY!"

How does everyone feel about adding some of my other suggestions from https://github.com/lbryio/types/issues/8 -at least Language (and/or Location? - that's what YT has) and Featured Video (lbry URL/claimid).

Reply:
* should `cover` and `thumbnail` be web URLs or LBRY URLs or claim IDs? - I think this should be similar to our current thumbnail system, and if we decide to support LBRY urls/hashes there, then we should here too. For now, http urls / spee.ch links should work fine.